REQUIRE

```diff
@@ -1,5 +1,6 @@
 julia 0.4
-DataArrays 0.3.4
+NullableArrays 0.0.5
+CategoricalArrays 0.0.2
```

Comment:
Not exactly sure if these are right or not, but `Pkg` seemed to be unhappy with my state of things when I checked out this branch.